frontend/src/metabase/components/Popover/constants.ts

```diff
@@ -1 +0,0 @@
-export const DEFAULT_Z_INDEX = 400;
```

Comment:
@rafpaf do we need to keep it?

Reply:
The only Z_INDEX value I care about is OVERLAY_Z_INDEX

It looks like I messed up, though. We have

```
frontend/src/metabase/css/core/overlays/constants.tsx
1:export const OVERLAY_Z_INDEX = 200;

frontend/src/metabase/css/core/overlays/constants.ts
1:export const OVERLAY_Z_INDEX = 200;
```

Could you please remove `frontend/src/metabase/css/core/overlays/constants.tsx` as part of this work?

Reply:
No problem if you don't want to (because of CI or some other reason)

Reply:
I wanted to, but I was afraid to do it, thanks for confirming!